src/stories/DatePicker.stories.js

```diff
@@ -8,11 +8,10 @@ import Row from '../components/Row';
 import Paragraph from '../components/Paragraph';
 import { mockPeriodOptions } from '../../test/mock/DatePicker';
 import theme from '../styles/theme';
-import AntdPicker from '../components/AntdPicker';
 
 export default {
   title: 'DatePicker',
-  component: AntdPicker,
+  component: DatePicker,
```

Comment:
Why do we remove the new DatePicker?

Reply:
From my understanding, the component people will use is DatePIcker, not the AntdPicker,which is used internally

Reply:
I guess you are right. At least for now, but in a near future the legacy component will be removed and the "AntdPicker" will be renamed as "DatePicker"